js_modules/dagster-ui/packages/ui-core/src/assets/useAllAssets.tsx

```diff
@@ -41,8 +42,14 @@ export function useAllAssets({
   batchLimit?: number;
 } = {}) {
   const client = useApolloClient();
-  const [materializedAssets, setMaterializedAssets] = useState<AssetRecord[]>([]);
   const manager = getFetchManager(client);
+  const [materializedAssets, setMaterializedAssets] = useState<AssetRecord[]>(() => {
+    const assetsOrError = manager.getAssetsOrError();
+    if (assetsOrError instanceof Array) {
```

Comment:
Slightly more reliable to do `Array.isArray`, I think?

Reply:
yeah thats true. If this were an array I'm receiving from a caller I'd use that but since its internal to this module and we know we're using the Array constructor it's not really necessary but I'll update it anyways